challenge_13/config.ru

```diff
@@ -20,7 +20,7 @@ class CharsetMiddleware
 
   def call(environment)
     response = Rack::Response[*@app.call(environment)]
-    response.content_type = "#{response.content_type}; charset=#{@charset}"
+    response.content_type = "#{response.content_type}; charset=#{@charset}" unless response.media_type_params.any?
```

Comment:
Great! Note that, in general, other media type parameters (e.g. [`boundary`](https://tools.ietf.org/html/rfc2046#section-5.1.1) in the case of `multipart/mixed`) might be present, so the _super_ correct thing here is to check for the presence of the `charset` parameter [specifically](https://github.com/adamluzsi/rack-charset/blob/v1.0.0/lib/rack/charset.rb#L16), but in practice other parameters are very rare so what you’ve done here will work fine.

Reply:
Ah, yep that makes sense! Thanks for making note of this ❤️